umm-spec-lib/resources/example-data/dif10/sample_collection.xml

```diff
@@ -139,5 +139,19 @@
       <Name>metadata.keyword_version</Name>
       <Value>8.4</Value>
     </Metadata>
-  </Extended_Metadata>
+    </Extended_Metadata>
+    <Extended_Metadata>
+    <Metadata>
+      <Group>gov.nasa.gsfc.gcmd.standardproduct</Group>
+      <Name>StandardProduct</Name>
+      <Value>true</Value>
+    </Metadata>
+    </Extended_Metadata>
+    <Extended_Metadata>
```

Comment:
why are these in here twice? Also, I think you can have multiple metadata tags in an extended_metadata tag.

Reply:
You have two in here to prove that it pulls the last I see, I retract my statement but ask that you confirm the code works with single extended_metatada tags containing multiple metadata tags.

Reply:
Added unit tests.